tests/kernel/gen_isr_table/src/main.c

```diff
@@ -28,7 +28,9 @@ extern u32_t _irq_vector_table[];
 #define TRIG_CHECK_SIZE		6
 #else
 #define ISR3_OFFSET	2
+#if !defined(CONFIG_SOC_SERIES_CC13X2_CC26X2)
```

Comment:
we should avoid having SoC/Board specific Kconfigs in tests, this will get very ugly when more SoCs/Boards are excluded like this. Please exclude it in testcase.yaml or actually we should whitelist this test just for boards that do not have conflicts in ISRs.

Reply:
I've also suggested that as an option.

However, I also see the point of testing as-much-as-we-can on each board. This test needs serious re-work anyways, eventually we will re-write this.

In brief, I am OK with both approaches.

Reply:
Since this is literally blocking #21326 (and #21326 in cascade) can I propose to merge this as is for now?

Reply:
@nashif @ioannisg I excluded the test from testcase.yaml as the simplest way to move this along, given the push back on having platform specific code. It is not entirely clear to me which boards need to be whitelisted, so I won't take that approach.

Reply:
@nashif FWIW I am ok with this change, this test honestly needs to be scrapped and rewritten